### PR TITLE
Changed asset page layout

### LINKED ISSUE #3769: Make Asset page compact
Asset page has a lot of white space currently and you need to scroll down to access the buttons.
TODO : shift buttons and warranty card to right side, reduce white space

### PR DESCRIPTION
**Pull request by skks1212**
closes #3769 
Reduces extra whitespace and makes layout compact. No need to scroll to access buttons.
Before:
image
After:
image

### CODE REVIEW

**Review by rithviknishad — COMMENTED** [review]
I tried a similar layout before and was asked to be changed here https://github.com/coronasafe/care_fe/pull/3458#issuecomment-1246458901
, and isn't there too much whitespace?


image

**Comment by shivankacker** [comment]
@rithviknishad the problem was I had to scroll down to see everything. : (
I'd say this layout is much more effective. And there is going to be some whitespace as there is not much data on the page. But we can utilize the area in the future for more details
🤔

**Comment by joelsajichacko** [comment]
###  I think we can best use the white space like this; (below image) by changing the layout a little bit
Link to the figma design; 
https://www.figma.com/file/fqDMgeN2m4TCSz5KobZaKW/CoronaSafe-Network-Care?node-id=1578%3A563


Assets Details page

**Comment by gigincg** [comment]
@skks1212

**Comment by rithviknishad** [comment]
@skks1212 can we have new tailwind components along with this PR?
so we could reuse these components in other places eventually with consistent design.

- [ ] `Note`
- [ ] `PropertyDetail` (or some better name 😅)
- [ ] `<ContactDisplay type="mail" | "phone">` links to tel or mailto (would be nice if it could automatically detect mail or phone type from value prop without needing to specify it 😅)
- [ ] `FeatureTag` for Active and Working so we could reuse it later in the facility home page redesign.
image

ideal component usage for note and property detail could look like this:
```jsx
<>
  <PropertyDetail   property="Service Details"   icon={<i ... />}   className="">
    {asset_data.service_date} {/* text styling inherited from PropertyDetail */}
  </PropertyDetail>

  <PropertyDetail   property="Notes"   icon={<i ... />}>
    <Note>
      {asset_data.notes} {/* text styling overridden by Note */}
    </Note>
  </PropertyDetail>
</>
```

cc: @gigincg

**Comment by shivankacker** [comment]
@gigincg @rithviknishad @joelsajichacko working on new changes, thanks!

**Comment by nihal467** [comment]
@skks1212 
image

the alignments are off, and it look really different from the design in the PR

**Comment by shivankacker** [comment]
@nihal467 yep, forgot to mark as WIP. 😅
Still working on it.

File: src/Components/Assets/AssetWarrantyCard_old.tsx [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+          </a>
+        ) : (
+          <span className="tracking-wide sm:text-sm text-xs text-gray-400">
+            No Support Email Provided
+          </span>
+        )}
+      </div>
+    </div>
+  );
+}
```
Comment (rithviknishad) [inline src/Components/Assets/AssetWarrantyCard_old.tsx]: Let's delete the old warranty card from the codebase.

```suggestion
```

**Review by rithviknishad — APPROVED** [review]

**Comment by nihal467** [comment]
LGTM

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**